hnn_core/params_default.py

```diff
@@ -359,3 +361,124 @@ def get_L5Pyr_params_default():
         'L5Pyr_dend_gbar_cat': 2e-4,
         'L5Pyr_dend_gbar_ar': 1e-6,
     }
+
+
+def _secs_L2Pyr():
```

Comment:
+1 for moving this code to `params_default.py` for now. What do you think of eventually converting this to NeuroML and storing the cell/network specifications in a standardized fashion? (I'm down to attempt this in a later PR if you think this would be a worthwhile endeavor.)

Reply:
If you really want to go down the NeuroML route, play around with the software first and share some scripts so we can be convinced :)